release_notes.md

```diff
@@ -1 +1 @@
-* Bug fix: Activity Functions can now use output bindings (https://github.com/Azure/azure-functions-powershell-worker/issues/646)
\ No newline at end of file
+* New feature: The `Get-TaskResult` CmdLet can now be used to get the result of already completed DF Tasks! Thanks @oobegreg for your contribution!
```

Comment:
This belongs to a different PR, right?

Reply:
Yes, I believe so. This diff should disappear once the other PR is merged.

Reply:
Yes, believe so. I think this diff will disappear once the PR for that feature is merged. I'll wait for that.